pkg/domain/stateDiffConfig.go

```diff
@@ -49,6 +52,20 @@ const (
 	ConfigActionRemove ConfigAction = "remove"
 )
 
+func determineConfigDiff(
+	blueprintConfig Config,
+	actualGlobalConfig map[common.GlobalConfigKey]ecosystem.GlobalConfigEntry,
+	actualDoguConfig map[common.DoguConfigKey]ecosystem.DoguConfigEntry,
+	actualSensitiveDoguConfig map[common.SensitiveDoguConfigKey]ecosystem.SensitiveDoguConfigEntry,
+) (CombinedDoguConfigDiff, GlobalConfigDiff, error) {
```

Comment:
🚫 **[checkstyle]** <unparam> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>determineConfigDiff - result 2 (error) is always nil